From: Michael Froman <mjfroman@mac.com>
Date: Fri, 2 May 2025 17:14:08 -0500
Subject: (tmp-cherry-pick) Revert "Checkout base/ only if fuzzing is enabled"
 (a337cab322)
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This reverts commit 924b73c1d3110d8dfd7cd5c9c60e94a6db0fcb46.

Reason for revert: This is breaking the DEPS update script (https://logs.chromium.org/logs/webrtc/buildbucket/cr-buildbucket/8721155299506494929/+/u/autoroll_DEPS/stdout)

Original change's description:
> Checkout base/ only if fuzzing is enabled
>
> Bug: webrtc:42223878
> Change-Id: I7d25551f8b82e84d4d123ef7768899342b87fde8
> Reviewed-on: https://webrtc-review.googlesource.com/c/src/+/380120
> Auto-Submit: Daniel.L (Byoungchan) Lee <daniel.l@hpcnt.com>
> Reviewed-by: Björn Terelius <terelius@webrtc.org>
> Commit-Queue: Mirko Bonadei <mbonadei@webrtc.org>
> Reviewed-by: Mirko Bonadei <mbonadei@webrtc.org>
> Cr-Commit-Position: refs/heads/main@{#44057}

Bug: webrtc:42223878
Change-Id: I7c871bdad972859a501d0db84263d2a7466bf8cf
Reviewed-on: https://webrtc-review.googlesource.com/c/src/+/381760
Commit-Queue: Jeremy Leconte <jleconte@webrtc.org>
Reviewed-by: Mirko Bonadei <mbonadei@webrtc.org>
Reviewed-by: Björn Terelius <terelius@webrtc.org>
Bot-Commit: rubber-stamper@appspot.gserviceaccount.com <rubber-stamper@appspot.gserviceaccount.com>
Cr-Commit-Position: refs/heads/main@{#44163}
---
 DEPS | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/DEPS b/DEPS
index f571093c94..df6d79bd56 100644
--- a/DEPS
+++ b/DEPS
@@ -62,10 +62,9 @@ vars = {
 }
 
 deps = {
-  'src/base': {
-    'url': 'https://chromium.googlesource.com/chromium/src/base@bf4cc197a277e3949b650c8f6f4f329bf261d700',
-    'condition': 'checkout_fuzzer'
-  },
+  # TODO(kjellander): Move this to be Android-only.
+  'src/base':
+    'https://chromium.googlesource.com/chromium/src/base@bf4cc197a277e3949b650c8f6f4f329bf261d700',
   'src/build':
     'https://chromium.googlesource.com/chromium/src/build@e7031cc6a0df7cf30e22f51b630b5f467b9fd7e2',
   'src/buildtools':
